### ISSUE TITLE
Missing alt text on contributors images

### ISSUE THREAD

**Issue by pettinarip** [ethereum/ethereum-org-website#5816]
**Describe the bug**
The contributors images don't have an `alt` text.

**To Reproduce**

1. Go to https://ethereum.org/en/contributing/#contributors
2. The contributors images don't have an `alt` text

**Expected behavior**

Images should have an `alt` text. E.g. `alt="Contributor {name}"`

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by wackerow** [comment]
Closed with above PR.
